lib/pbench/test/unit/server/auth/conftest.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import jwt
+import pytest
+
+from pbench.server.auth import OpenIDClient
+
+
+def mock_set_oidc_auth_endpoints(oidc_client):
+    oidc_client.USERINFO_ENDPOINT = "https://oidc_userinfo_endpoint.example.com"
+    oidc_client.JWKS_ENDPOINT = "https://oidc_jwks_endpoint.example.com"
+
```

Comment:
In your last update, you added `TOKENINFO_ENDPOINT` back to the `OpenIDClient` object...I don't know whether you want to be leaving it as `None` here or not.  Realistically, we should probably have tests for all of the interesting combinations of endpoints being present and absent.